.travis.yml

```diff
@@ -26,6 +32,8 @@ before_install:
   - travis_retry composer self-update && composer --version
 
 install:
+  - if [ "$SYMFONY_VERSION" != "" ]; then travis_retry composer require "symfony/framework-bundle:${SYMFONY_VERSION}" "symfony/dependency-injection:${SYMFONY_VERSION}" --no-update -n; travis_retry composer require "symfony/twig-bundle:${SYMFONY_VERSION}" --no-update -n; fi;
+  - if [ "$SYMFONY_VERSION" != "" ]; then travis_retry composer require "symfony/symfony:${SYMFONY_VERSION}" --prefer-source --no-update -n; fi;
```

Comment:
Woops!

I'm hesitant about the `travis_retry` here as well. This was copied from other examples, but I'll just remove it here for now.